docs/resources/management_access_policy.md

```diff
@@ -0,0 +1,528 @@
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  ssh_service = {
+    admin_state   = "disabled"
+    annotation    = "annotation_1"
+    description   = "description_1"
+    hostkey_algos = ["rsa-sha2-256"]
+    kex_algos     = ["curve25519-sha256"]
+    name          = "name_1"
+    name_alias    = "name_alias_1"
+    password_auth = "disabled"
+    port          = "22"
+    ssh_ciphers   = ["aes192-ctr"]
+    ssh_macs      = ["hmac-sha1"]
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Management Access Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_management_access_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Management Access Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Management Access Policy object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the Management Access Policy object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Management Access Policy object.
+* `name_alias` (nameAlias) - (string) The name alias of the Management Access Policy object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `strict_security_on_apic_oob_subnet` (strictSecurityOnApicOOBSubnet) - (string) remove implicit access on apic oob from any apic OOB subnet IP.
+  - Default: `false`
+  - Valid Values: `no`, `yes`.
+  
+* `http_service` - (map) A map of Http Service (ACI object [commHttp](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/commHttp/overview)).
+  #### Optional ####
+    
+  * `access_control_allow_credential` (accessControlAllowCredential) - (string) Determines if credentials are allowed in HTTP responses.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+  * `access_control_allow_origins` (accessControlAllowOrigins) - (string) Specifies which origins can access the resource.
+  * `admin_state` (adminSt) - (string) The administrative state of the HTTP Service.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+  * `annotation` (annotation) - (string) The annotation of the Http Service object.
+      - Default: `orchestrator:terraform`
+  * `cli_only_mode` (cliOnlyMode) - (string) Indicates if CLI-only mode is enabled.
```

Comment:
should we standardise on determines vs specifies vs indicates?

Reply:
Ok